Week2/js-exercises/ex1-oddOnesOut.js

```diff
@@ -7,15 +7,13 @@
  The function should still behave the same.
 
  */
-function doubleEvenNumbers(numbers) {
-  const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
-  return newNumbers;
-}
 
 const myNumbers = [1, 2, 3, 4];
-console.log(doubleEvenNumbers(myNumbers)); // Logs "[4, 8]" to the console
\ No newline at end of file
+
+let doubleEvenNumbers =(numbers) => {
+    let newNumbers = numbers.filter(numbers => (numbers % 2 == 0))
+    newNumbers = newNumbers.map(numbers => numbers * 2 )
+    return newNumbers
+  };
```

Comment:
This definitely works! Something to consider is the reassignment of the variable newNumbers. You could also chain the array functions and not use variables at all:
```suggestion
let doubleEvenNumbers = numbers => numbers
    .filter(numbers => numbers % 2 === 0)
    .map(numbers => numbers * 2 )
```

Reply:
Thanks. I'll definitely consider the reassignment of the variables. Chaining the array functions totally makes sense and I started practicing it by JS3.